docker-compose.yml

```diff
@@ -140,16 +140,29 @@ services:
       - ./netkan:/home/netkan/netkan
     command: download-counter
   dev:
+    user: netkan
     build:
       context: netkan/.
       target: dev
     environment:
       SSH_KEY: ${CKAN_NETKAN_SSHKEY}
-      NETKAN_REMOTE: ${NETKAN_METADATA_PATH}
       CKANMETA_REMOTE: ${CKAN_METADATA_PATH}
+      CKANMETA_USER: ${CKAN_METADATA_USER}
+      CKANMETA_REPO: ${CKAN_METADATA_REPO}
+      NETKAN_REMOTE: ${NETKAN_METADATA_PATH}
+      AWS_DEFAULT_REGION: ${CKAN_AWS_DEFAULT_REGION}
+      AWS_SECRET_ACCESS_KEY: ${CKAN_AWS_SECRET_ACCESS_KEY}
+      AWS_ACCESS_KEY_ID: ${CKAN_AWS_ACCESS_KEY_ID}
       GH_Token: ${CKAN_GH_Token}
+      SQS_TIMEOUT: 30
+      STATUS_DB: DevNetKANStatus
+      XKAN_GHSECRET: test
+      INFLATION_SQS_QUEUE: InboundDev.fifo
+      MIRROR_SQS_QUEUE: MirroringDev.fifo
+      STATUS_BUCKET: ckan-test-status
+      STATUS_INTERVAL: 0
       DISCORD_WEBHOOK_ID: ${DISCORD_WEBHOOK_ID}
       DISCORD_WEBHOOK_TOKEN: ${DISCORD_WEBHOOK_TOKEN}
     volumes:
-      - ./netkan:/home/netkan/netkan
+      - .:/home/netkan/workspace
     entrypoint: sleep infinity
```

Comment:
`docker-compose.yml` isn't used by VS Code, it's for spinning up a full integration test. It admittedly needs more work and documentation. We should possibly move the `docker-compose.vscode.yml` under `.vscode` and reference it from there.

Reply:
But it does basically the same, doesn't it? It starts up the container and goes to idle, waiting for a user to open a remote console for testing stuff?

Reply:
Yeah, I added it before I implemented all the bits and pieces for VS Code. This stub here will never get used by VS code. But they share a project namespace for convenience.

Reply:
Is it used by anything else? Should I delete it entirely?

Reply:
I don't think we need to stress too hard on this PR. We probably should have a rethink of the dev workflow, since you'll be more likely to use this I'm happy for you to take the lead.

Reply:
So just to make sure, this is not used by you (or @HebaruSan?) or any testing stuff?

Because if not, I'd remove it, I don't need it either. But I don't want to break anyones workflow there.

Reply:
I have no strong opinions either way. The infrastructure is bedded down enough where this probably isn't needed and anything else we do should really be adding tests.

Reply:
If I'm using this somehow, I don't know about it. I'm very much following the lead of whatever you folks put in the files when it comes to docker.